TeslaCybertruck.Shared/MainPage.xaml.cs

```diff
@@ -41,6 +41,20 @@ public bool SecondGlassBroken
 			set => SetValue(SecondGlassBrokenProperty, value);
 		}
 
+        private async void CustomizeIt_Click(object sender, RoutedEventArgs e)
+        {
+            try
+            {
+                await Windows.System.Launcher.LaunchUriAsync(new Uri("https://playground.platform.uno/#cybertruck"));
+            }
+            catch
+            {
+                // Oops
```

Comment:
Can we have something else here ;) ?

Reply:
We could :D But we don't have loggers configured and I don't think it's worth it. I doubt it will ever fail since the uri is hardcoded. It would only fail if uri is invalid or we're not on the main thread.